frontend/pages/pricing.tsx

```diff
@@ -82,7 +83,7 @@ export default function Pricing() {
                     </Box>
 
                     <VStack
-                        bg={useColorModeValue('gray.50', 'gray.700')}
+                        bg={useColorModeValue('white', 'gray.700')}
```

Comment:
@TheAayushRana to understand why there are two colours here, the second one is for dark mode (it is not activate in lupax)